server/go/api_static_routes_service_impl.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * Kusk Gateway API
+ *
+ * This is the Kusk Gateway Management API
+ *
+ * API version: 1.0.0
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package openapi
+
+import (
+	"context"
+	"net/http"
+
+	kusk "github.com/GIT_USER_ID/GIT_REPO_ID/kusk"
+)
+
+// StaticRoutesApiService is a service that implements the logic for the StaticRoutesApiServicer
+// This service should implement the business logic for every endpoint for the StaticRoutesApi API.
+// Include any external packages or services that will be required by this service.
+type StaticRoutesApiService struct {
+	kuskClient kusk.Client
+}
+
+// NewStaticRoutesApiService creates a default api service
+func NewStaticRoutesApiService(kuskClient kusk.Client) StaticRoutesApiServicer {
+	return &StaticRoutesApiService{kuskClient: kuskClient}
+}
+
+// GetStaticRoute - Get details for a single static route
+func (s *StaticRoutesApiService) GetStaticRoute(ctx context.Context, namespace string, name string) (ImplResponse, error) {
+	staticRoute, err := s.kuskClient.GetStaticRoute(namespace, name)
+	if err != nil {
+		return Response(http.StatusInternalServerError, err), err
+	}
+	return Response(http.StatusOK, StaticRouteItem{
+		Name:      staticRoute.Name,
+		Namespace: staticRoute.Namespace,
+	}), nil
+}
+
+// GetStaticRouteCRD - Get static route CRD
+func (s *StaticRoutesApiService) GetStaticRouteCRD(ctx context.Context, namespace string, name string) (ImplResponse, error) {
+	staticRoute, err := s.kuskClient.GetStaticRoute(namespace, name)
```

Comment:
> staticRoute, err := s.kuskClient.GetStaticRoute(namespace, name)

Should this be called `GetStaticRouteCRD(...)` ?

Reply:
Not really. The only difference is that here we don't do any operations on fleet object we just return it at the end of the method.

Reply:
Im not sure I understand, the Service method is called `GetStaticRouteCRD` but inside we call the kuskClient `GetStaticRoute`? that's the correct behaviour?